packages/writing-style/package.json

```diff
@@ -27,5 +27,8 @@
     "7zip-bin": "5.1.1",
     "node-fetch": "2.6.7",
     "shelljs": "0.8.5"
+  },
+  "devDependencies": {
+    "prettier": "2.6.1"
```

Comment:
I've installed prettier in order to use the two scripts for updating the google and writegood styles

Reply:
Interesting, I previously thought it's sufficient to have a CLI in the top level package.json to be able to use it in a workspace context.